spec/integrations/groups_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Group', type: :feature do
+  describe 'new' do
+    before(:each) do
+      User.destroy_all
+      Group.destroy_all
+      @user = User.create(name: 'Atif', email: 'atif@email.com', password: 'asd12345')
+      @group = Group.create(name: 'easypaisa', icon: Rack::Test::UploadedFile.new('spec/media/easypaisa.png'))
+      visit new_user_session_path
+      fill_in 'Email', with: 'atif@email.com'
+      fill_in 'Password', with: 'asd12345'
+      click_button 'Log in'
+    end
+    it 'Should have data' do
+      expect(page).to have_content('Your Catogeries')
+      expect(page).to have_content('Back')
+    end
+    it 'New group shouild be added' do
+      click_link 'Add New Category'
+      expect(page).to have_content('Name')
```

Comment:
You have some failing tests it would be nice if you fixed the failing test cases. 

Take a look at this screen shot:
![Screenshot 2023-08-11 at 3 37 55 PM](https://github.com/Atifzada/cash-nest/assets/79506105/57c5f2cd-eab3-424a-b6f5-6d0ea2db47d8)

Reply:
Thank you for your review. But I want to add that all the tests were clearing at my end. Obviously I will look into it and will come up with solution